Backend/.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
```

Comment:
We should use the same standard license header used in the CPP files of JasmineGraph project here

Reply:
sure, i added standard license header used in the CPP files of JasmineGraph project

Reply:
Following file still missing license header,

Frontend/src/app/clusters/[id]/logs/page.tsx

Reply:
added

Reply:
Ok